nvflare/app_common/workflows/scatter_and_gather.py

```diff
@@ -234,14 +234,14 @@ def control_flow(self, abort_signal: Signal, fl_ctx: FLContext) -> None:
 
                 self.fire_event(AppEventType.BEFORE_SHAREABLE_TO_LEARNABLE, fl_ctx)
                 self._global_weights = self.shareable_gen.shareable_to_learnable(aggr_result, fl_ctx)
-                fl_ctx.set_prop(AppConstants.GLOBAL_MODEL, self._global_weights, private=True, sticky=True)
+                fl_ctx.set_prop(AppConstants.GLOBAL_MODEL, self._global_weights, private=True, sticky=False)
```

Comment:
same here, scatter_and_gather is used much more often for other pipelines, not sure how much impact sticky has @YuanTingHsieh

Reply:
let merge this and I will do some testing on other examples to see if this is fine or not.

Thanks